client/web-sveltekit/src/lib/repo/open-in-editor/EditorIcon.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+    import type { Editor } from '$lib/web'
+    export let editorId: Editor['id']
+
+    let Icon: any;
+    $: import(`./editors/${editorId}.svelte`).then(i => Icon = i.default);
+</script>
+
+ {#if Icon}
+    <Icon />
+ {/if}
```

Comment:
I'm wondering whether this works:

```suggestion
</script>

 {#await import(`./editors/${editorId}.svelte`) then {default: Icon}}
    <Icon />
 {/await}
```

(or something like that)

Reply:
It does! I like that :)